wordpress-mobile/WordPress-Android
# WordPress Stories : Story title bottom sheet integration part 1

Fixes #12305
## Solution
To integrate the existing prepublishing bottom sheet with the Story Composer. Currently, this solution hardcodes the story title UI within the bottom sheet for pages and post just so this PR can be tested. In future PRs, the entry point will be from the Story Composer. 

## Testing
1. Create a new post. 
2. Click publish. 
3. See that the bottom sheet UI asks for a story title and you are able to enter one. 

Before | After 
--------|-------
 <kbd><img src="https://user-images.githubusercontent.com/1509205/85951489-1e37ea00-b929-11ea-99da-6dc392b56df0.png" width="320"></kbd>       |       <kbd><img src="https://user-images.githubusercontent.com/1509205/85951486-1d06bd00-b929-11ea-8ebd-f214ee977b58.png" width="320"></kbd>

## Concerns
1. Currently, even though I am requesting the story title's edit text focus and showing the keyboard it still doesn't show up initially. 
2. The story title UI might need some tweaking but keeping it as is for now, and will improve on it in future PRs. 

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
